libyul/optimiser/Suite.cpp

```diff
@@ -112,12 +112,18 @@ void OptimiserSuite::run(
 	// None of the above can make stack problems worse.
 
 	size_t codeSize = 0;
-	for (size_t rounds = 0; rounds < 30; ++rounds)
+	size_t sizeStableForRounds = 0;
```

Comment:
@cameel can this part salvaged? Does the optimiser suite does something like this now?

Reply:
Sure. This part was simply moved to a function called [runSequenceUntilStable()](https://github.com/ethereum/solidity/blob/develop/libyul/optimiser/Suite.cpp#L327-L343) and the functionality was left the same. You can just copy your change there.

Reply:
Oh, and I made the number of rounds a constant: [OptimiserSuite::MaxRounds](https://github.com/ethereum/solidity/blob/develop/libyul/optimiser/Suite.h#L48)

Reply:
I think this is different from `runSequenceUntilStable`.

Reply:
Yeah, it's different but I thought @axic was asking if it's possible to easily apply these changes on top of the current state of `OptimiserSuite`.